### ISSUE TITLE
Where to go after clicking 'no' on 'target-ad' page

### ISSUE THREAD

**Issue by dhowe** [dhowe/spectre#362]
currently just continues to '/success-ad'

**Comment by billposters** [comment]
Thanks for raising this one. we created some alternative routes to 'nudge' the user if they select no for other choices but not to the best of my knowledge for this page. Do you think we should create something similar? But keep additional page # low?

**Comment by dhowe** [comment]
Maybe 1-2 additional pages at most (I think our page count is still way too high)... though low priority

**Comment by billposters** [comment]
Yeah I agree. Some of our initial research on framing / content for alternative 'no' routes is up here: https://www.figma.com/file/K4rxWKtuG6abqJZkSzIq4yWm/SPECTRE---UX--FINAL-2?node-id=1%3A5177

**Comment by billposters** [comment]
Do we have a decision on the content for a 'no' route?
I would like to use the text that gives the user a friendly 'nudge' to say say. 

Example: 

'Don't worry! Facebook allows anyone with enough money to micro-target people with consumer and politcal ads. It is perfectly normal! 

[Share your ad]  [no thanks, I'm ok!]

**Comment by dhowe** [comment]
Sure. Just need a design for the page

**Comment by billposters** [comment]
ok can you add Pete Cadney to this github so we can ensure he is aware of this spec for an additional page or 2?

**Comment by dhowe** [comment]
need an email

**Comment by billposters** [comment]
hello@petercadney.co.uk

**Comment by dhowe** [comment]
An email address with a github account

**Comment by billposters** [comment]
please ask pete!

**Comment by billposters** [comment]
thanks!
